docs/en/writing-running-appium/caps.md

```diff
@@ -165,7 +165,51 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 |Capability|Description|Values|
 |----|-----------|-------|
 |`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for an espresso server to launch. Defaults to `30000` |e.g., `50000`|
+|`espressoBuildConfig`|Path to an espresso server build configuration JSON (see below)|e.g., `/projects/myapp-tests/buildconfig.json`|
+|`showGradleLog`|Always output Gradle build log for an espresso server in the server log. Defaults to `false`|e.g., `true`|
 
+##### Espresso server build configuration JSON
+Passing this configuration file using `espressoBuildConfig` desired capability allows changing how an espresso server is built. It is mostly useful in cases where the default espresso server settings are not compatible with your application under test.
```

Comment:
How about adding a link to https://github.com/appium/appium-espresso-driver/issues/449 as the typical example?

Reply:
Would this phrasing be good?
> One example of such a case is tests crashing due to `Resource <name> is not a Drawable` error (see https://github.com/appium/appium-espresso-driver/issues/449 for discussion).

Also do you want it to be added to some troubleshooting page (which)?